### ISSUE TITLE
Stage 4/4: Model Evaluation 

### ISSUE THREAD

**Issue by Carmoruda** [Carmoruda/JetBrains-Academy-Projects#17]
## To-do list
- [ ] Import the evaluation metrics library
- [ ] Predict the model output for the test dataset. Print the predictions
- [ ] Print the confusion matrix using the `sklearn` library. More on `confusion_matrix` can be found in the [Sklearn documentation](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)
- [ ] Print the score of the model.

#### Description
Once you have a model based on the logistic regression, it’s the best practice to evaluate it and check how it performs against other models.

During an evaluation with logistic regression models, accuracy checks may give a false account of what is happening inside the model. One way to evaluate logistic regression properly is by checking for **True Positives** (TP), **False Negatives** (FN), **False Positives** (FP), and **True Negative** (TN). A **confusion matrix** can help you with that.

A confusion matrix is used when the output variable has two or more classes. This metric is also good for measuring **precision**, **recall**, **specificity**, **accuracy**, and **f-scores**.

At this stage, we will predict the result of the model and evaluate its quality using metrics from `sklearn` library.
### Objectives
To complete this stage, you should:

1.  Import the evaluation metrics library;
2.  Predict the model output for the test dataset. Print the predictions;
3.  Print the confusion matrix using the `sklearn` library. More on `confusion_matrix` can be found in [the Sklearn documentation](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html);
4.  Print the score of the model.
#### Examples
**Example 1**

*Input: the program should accept the x_test data set that wasn't trained*

*Output: the predicted values given by the algorithm*

```Python
[0 0 0 ... 0 0 0]
```
**Example 2**

*Input: the program should accept the predicted values and the target class*

*Output: the number of correct predictions*

```Python
array([[56865,    10],
       [   39,    48]])
```
**Example 3**

*Input: the program should accept the training datasets*

*Output: the program should display the score of the model*

```Python
0.99923632293884
```
